packages/api/src/beacon/routes/events.ts

```diff
@@ -5,18 +5,6 @@ import {RouteDef, TypeJson} from "../../utils/index.js";
 
 // See /packages/api/src/routes/index.ts for reasoning and instructions to add new routes
 
-export type LightclientOptimisticHeaderUpdate = {
```

Comment:
This was removed in other to use the ssz types directly...instead of redefining the same types. Is there a reason not to remove and have the types redefined?

Reply:
No thats perfect. I think they were "temporarily" and then forgotten about